peps/pep-0731.rst

```diff
@@ -4,13 +4,14 @@ Author: Guido van Rossum <guido@python.org>,
         Petr Viktorin <encukou@gmail.com>,
         Victor Stinner <vstinner@python.org>,
         Steve Dower <steve.dower@python.org>,
-        Irit Katriel <irit@python.org>
+        Erlend Aasland <erlend@python.org>
```

Comment:
should irit be kept as author?

Reply:
Yeah, makes sense.

```suggestion
        Irit Katriel <irit@python.org>,
        Erlend Aasland <erlend@python.org>
```

Reply:
I'm not sure it makes sense to add me to the authors list, though.

Reply:
My inclination would be no: the authors are the people who originally wrote the PEP, so the author list should remain static.

Reply:
Though as this is a Process PEP, it could receive significant changes later; if so, a new author could be added. But that's not the case here.

Reply:
Totally agree; I'll push an update.

Reply:
Resolved with 7e72c70a743835c74c89752c8f862f042001ae1a